examples/MomentoFSharpApplication/Program.fs

```diff
@@ -19,9 +22,11 @@ let exerciseCache() = (
 
         printfn("Listing caches:")
         let resp = client.ListCaches()
-        for cacheInfo in resp.Caches do
-            printfn $"{cacheInfo.Name}"
-        
+        let _ = match resp with
+                | :? ListCachesResponse.Success as successResponse ->
+                    for cacheInfo in successResponse.Caches do
+                        printfn $"{cacheInfo.Name}"
+                | _ -> ()
```

Comment:
:heart_eyes:

Reply:
Yesterday was my first encounter of F# 😱